subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -610,10 +663,19 @@ void ztest_verify_all_test_suites_ran(void)
 	}
 }
 
+void ztest_run(const void *state)
```

Comment:
I like the idea of exposing this so that others can leverage it without having to replicate the logic. Can we add the function signature to the header? Maybe rename to `test_main_default()` or some variant of that?

Reply:
Done